cloud_testing/kotlin_poc/src/main/kotlin/ftl/gc/GcAndroidTestMatrix.kt

```diff
@@ -70,7 +70,7 @@ object GcAndroidTestMatrix {
         testMatrix.testSpecification = TestSpecification()
                 .setAndroidInstrumentationTest(androidInstrumentation)
                 .setDisablePerformanceMetrics(config.disablePerformanceMetrics)
```

Comment:
The negatives mean this does the opposite of what's intended.

`performance-metrics` set to true means we want to enable performance metrics. `performance-metrics` is stored as `disablePerformanceMetrics` which is passed to `setDisablePerformanceMetrics`. Enabling performance metrics disables them. 🙂 

we should probably do something like this: `.setDisablePerformanceMetrics(!config.performanceMetrics)`